src/domain/Avatars/Avatar/Avatar.scss

```diff
@@ -33,7 +33,6 @@ $status-dot-styles: (
 }
 
 :local(.Avatar) {
-  display: inline-block;
```

Comment:
will it affect the existing avatar?

Reply:
I didn't see any change with any of the avatar components